src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,11 @@
+.main__catalog {
+  margin: 0 auto;
+  padding: 50px 0;
+  width: $catalog-width;
+}

```

Comment:
each element should be nested into it's block

Reply:
changed